src/main/java/com/amazonaws/services/sqs/AmazonSQSIdleQueueDeletingClient.java

```diff
@@ -104,13 +104,19 @@ private QueueMetadata(String name, String queueUrl, Map<String, String> attribut
     private IdleQueueSweeper idleQueueSweeper;
     private String deadLetterQueueUrl;
 
-    public AmazonSQSIdleQueueDeletingClient(AmazonSQS sqs, String queueNamePrefix) {
+    public AmazonSQSIdleQueueDeletingClient(AmazonSQS sqs, String queueNamePrefix, Long heartbeatInterval) {
         super(sqs);
         
         if (queueNamePrefix.isEmpty()) {
             throw new IllegalArgumentException("Queue name prefix must be non-empty");
         }
         this.queueNamePrefix = queueNamePrefix;
+        this.HEARTBEAT_INTERVAL_SECONDS = heartbeatInterval != null ? heartbeatInterval : 5;
```

Comment:
Can you define the default as a constant in the builder and reference it here instead of duplicating? I get that you're maintaining the constructor signature just to be safe.

Reply:
Moved the default into the builder.